Tests/scripts/utils/get_modified_files_for_testing.py

```diff
@@ -17,7 +17,7 @@
 
 class FileType(constants.FileType, Enum):
     CONF_JSON = "confjson"
-    METADATA = "metadata"
+    SECRETS_IGNORE = "secretsignore"
```

Comment:
why do we need it? this file should not affect any logics related to selected tests/packs